tests/expressions/tests.py

```diff
@@ -381,6 +381,24 @@ def test_object_update_unsaved_objects(self):
         with self.assertRaisesMessage(ValueError, msg):
             acme.save()
 
+    def test_ticket_30129_insert_f_expression_in_subquery(self):
```

Comment:
Ticket numbers are unnecessary in the test method names:
```suggestion
    def test_insert_f_expression_in_subquery(self):
```